### DISCUSSION TITLE
display single topic over time

### DISCUSSION THREAD

**Discussion by aminathshausan** [MaartenGr/BERTopic#780] [Q&A] [answered]
Hi, 

Is there a way to view a single topic over time? At the moment I am using the following code, but it displays all topics. 
```
topics_over_time = topic_model.topics_over_time(abstracts,  timestamps)
topic_model.visualize_topics_over_time(topics_over_time, top_n_topics=11, width=900, height=500)
```
Thanks

**Comment by MaartenGr** [comment, marked as answer]
Yes, there is the `topics` parameter in ` .visualize_topics_over_time` that you can select to display a single topic.
